src/OnboardingSPA/steps/SiteGen/SiteDetails/contents.js

```diff
@@ -20,6 +20,17 @@ const getContents = () => {
 			'wp-module-onboarding'
 		),
 		walkThroughlink: __( 'click here', 'wp-module-onboarding' ),
+		languageList: [
+			[ __( 'English (US)', 'wp-module-onboarding' ), 'en-US' ],
+			[ __( 'English (UK)', 'wp-module-onboarding' ), 'en-UK' ],
+			[ __( 'Spanish', 'wp-module-onboarding' ), 'es-ES' ],
+			[ __( 'French', 'wp-module-onboarding' ), 'fr-FR' ],
+			[ __( 'Hindi', 'wp-module-onboarding' ), 'hi-IN' ],
+		],
```

Comment:
We need to support more languages than just what's on this list. I think we can use `wp_dropdown_languages()` and feed that from the backend. see this internal item for more info: https://jira.newfold.com/browse/PRESS8-324